nested.just

```diff
@@ -9,6 +9,7 @@ export taskPath := invocation_directory()
 export councilSafe := env_var("COUNCIL_SAFE")
 export foundationSafe := env_var("FOUNDATION_SAFE")
 export ownerSafe := env_var('OWNER_SAFE')
+export chainGovernorSafe := env_var('CHAIN_GOVERNOR_SAFE')
```

Comment:
Then in the justfile we should error if they pass `chain-governor` as the safe kind but don't specify the env var
```suggestion
export chainGovernorSafe := env_var_or_default('CHAIN_GOVERNOR_SAFE', '')
```

Reply:
sg, this is an example of it failing when a user passes chain-governor without setting the var.
```
blaine@blaines-MacBook-Pro 018-granite-upgrade % just --dotenv-path $(pwd)/.env --justfile ../../../nested.just simulate chain-governor            
Running script with assertions
Using script /Users/blaine/code/superchain-ops/tasks/eth/018-granite-upgrade/NestedSignFromJson.s.sol
getting signer address for chain-governor...
Error: CHAIN_GOVERNOR_SAFE is not set for chain-governor.
error: Recipe `simulate` failed with exit code 1
```

this was implemented in commit: https://github.com/ethereum-optimism/superchain-ops/pull/370/commits/23cac55edf801b8c8684f190da0c6025bc0c7732